scanner/inventory_test.go

```diff
@@ -265,22 +265,6 @@ func TestFindings(t *testing.T) {
 				Details: "Used in 1 repo(s)",
 			},
 		},
-		{
-			RuleId: "known_vulnerability_in_build_platform",
-			Purl:   "gitlab",
-			Meta: opa.FindingMeta{
-				OsvId:   "CVE-2024-2651",
-				Details: "Provider: gitlab",
-			},
-		},
-		{
-			RuleId: "known_vulnerability_in_build_platform",
-			Purl:   "gitlab",
-			Meta: opa.FindingMeta{
-				OsvId:   "CVE-2024-5655",
-				Details: "Provider: gitlab",
-			},
-		},
```

Comment:
hmmmm so we have zero test for this now ?
I can find `CVE-2024-2651` in the file and it's `>=0.0,<16.9.7`, so should still match !?

Reply:
@fproulx-boostsecurity that version added 80 CVEs to the test and everytime we update it will add more so for the sake of maintenance I removed it

Reply:
@SUSTAPLE117 I see, maybe we can find a more recent version that has just like one or two.... does not have to be Gitlab, but at least so we have a basic test